My fourth pull request

JustSanya123 opened the pull request:
<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

mzjp2 committed:
Correct typos in Windows installation guide for Go (#19991)

"Windodows" -> "Windows" in two occurrences and "ok" -> "okay" for stylistic reasons.

Commit:
fix typo in vuejs intro (#20092)

JustSanya123 commented:
Pls don`t block me if i did something wrong

ArielLeslie commented:
You checked a box indicating "My pull request has a descriptive title (not a vague title like `Update index.md`)", but you have given it the vague/meaningless title "my fourth pull request". Descriptive titles and good explanations of changes are necessary for a well maintained codebase. Free Code Camp is choosing to relax some of its requirements to help people reach their Hacktoberfest goals, but this PR would normally be rejected. In the future, please help repository maintainers by providing useful titles and do not check off requirements without reading and following them.

Commit:
Update alt text challenge translation to pt-br (#19370)

wolfwhocodes committed:
Array variables have same data type (#23142)

An array is a set of homogeneous data. It's important to note that the variables in an array are always of the same data type.

csteacher1 committed:
Correct spelling fo "developer" (#23179)

In the sentence beginning on Line 15, "developer" is misspelled as "developper" and should be corrected.

akshay11298 committed:
Definition change (#24081)

catcecilia committed:
formatting change for #2 (#24080)

catcecilia committed:
Update index.md (#24076)

change in grammar and word-choice

sarahbe committed:
Add Arabic translation for Insertion Sort (#24075)

BubbaBeans committed:
Update index.md (#21047)

Cleaned up language structure a bit, filled out the definition.

thinkinbee committed:
Add MapReduce description to core hadoop (#24073)

lootster committed:
Fixed typo error on line 10 "Non-opinionated" (#24068)

nicoletlopez committed:
Updated "More Information" Link at Bootstrap Collapse (#24067)

Updated it to v4.1

aosimeon committed:
Update index.md (#21043)

Added comment at the beginning of the css code.

seand52 committed:
Update index.md (#24066)

Improved explanation of arrow function syntax

darrylyeo committed:
Fix typo: "Excercise" => "Exercise" (#24064)

fatima12nm committed:
Small Change on Accents (#21042)

darrylyeo committed:
Fix typo: "Excercise" => "Exercise" (#24063)

markoch committed:
Use a descriptive text for the Wikipedia link (#24057)

darrylyeo committed:
Fix typo: "reaplce" => "replace" (#24056)

Commit:
Updated syntax and reworked layout (#21040)

blaiseAI committed:
added " C programming ressources link" (#21039)

* added " C programming ressources link"

This links to TutorialsPoint which is a good ressource for most of the programming languages

* Added "Unordered list for the tutorial element"

novemberhaiku committed:
Update index.md (#21037)

Clarified some grammar and style

IshanFx committed:
Bound service (#24052)

According to the android guideline there are there services type and I added the bound service to this document to make it compatible with latest android guideline

CamB95 committed:
Update index.md (#24960)

* Update index.md

added Phaser to more info.

* Improvements to Game Engine links

Asjas commented:
This PR is going to merge `master` into the `tchaffee-patch-3` branch. I'm marking this as `invalid`
